pkg/resolver/searchSubscription.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Contributors to the Open Cluster Management project
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/stolostron/search-v2-api/graph/model"
+)
+
+func SearchSubscription(ctx context.Context, input []*model.SearchInput) (<-chan []*SearchResult, error) {
+	ch := make(chan []*SearchResult)
+
+	// You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
+	// For this example we'll simply use a Goroutine with a simple loop.
+	go func() {
+		// Handle deregistration of the channel here. Note the `defer`
+		defer close(ch)
+
+		for {
+			// Send the search results every 10 seconds.
+			time.Sleep(10 * time.Second)
```

Comment:
Let's make this polling interval configurable.
You can define a new env in `/pkg/config/config.go`

I'm also wondering if we should allow the client to override when creating the subscription. For this case maybe it would make sense to have a `MIN_SUBSCRIPTION_REFRESH_INTERVAL` and we allow the client to extend it. **This part can be done in the next iteration.**